### PR TITLE
bump CLI version to 2.1.0

### PR DESCRIPTION
**Pull request by 0xjinn**

### CODE REVIEW

File: crates/aptos/CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## Unreleased
```
Comment (gregnazario) [inline crates/aptos/CHANGELOG.md:5]: I would leave the `Unreleased` so people don't forget about it.

File: crates/aptos/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
...
+## [2.1.0] - 2023/08/21
+### Updated
 - Updated CLI source compilation to use rust toolchain version 1.71.1 (from 1.71.0).
+### Added
+- Added basic ledger support for CLI
+  - Example: `aptos init --ledger` to create a new account from ledger
```
Comment (gregnazario) [inline crates/aptos/CHANGELOG.md:10]: Maybe we should say you can do `aptos init --ledger` to create a profile from ledger, and from then on you can use it the same as any other profile?

Comment (0xjinn) [reply crates/aptos/CHANGELOG.md:10]: Good call!

File: crates/aptos/CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,13 @@
 All notable changes to the Aptos CLI will be captured in this file. This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and the format set out by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+
+## [2.1.0] - 2023/08/21
+### Updated
 - Updated CLI source compilation to use rust toolchain version 1.71.1 (from 1.71.0).
+### Added
+- Added basic ledger support for CLI
+  - Example: `aptos init --ledger` to create a new profile from ledger. After this, you can use it the same way as other normal profile.
```
Comment (gregnazario) [inline crates/aptos/CHANGELOG.md:12]:
```suggestion
  - Example: `aptos init --ledger` to create a new profile from ledger. After this, you can use it the same way as other profiles.
```

**Review by gregnazario — APPROVED** [review]

**Review by xbtmatt — APPROVED** [review]

**Pull request merged**